src/server.h

```diff
@@ -181,33 +181,34 @@ extern int configOOMScoreAdjValuesDefaults[CONFIG_OOM_COUNT];
 #define CMD_ASKING (1ULL<<13)          /* "cluster-asking" flag */
 #define CMD_FAST (1ULL<<14)            /* "fast" flag */
 #define CMD_NO_AUTH (1ULL<<15)         /* "no-auth" flag */
+#define CMD_CAN_REPLICATE (1ULL<<16)   /* "can-replicate" flag */
 
 /* Command flags used by the module system. */
-#define CMD_MODULE_GETKEYS (1ULL<<16)  /* Use the modules getkeys interface. */
-#define CMD_MODULE_NO_CLUSTER (1ULL<<17) /* Deny on Redis Cluster. */
+#define CMD_MODULE_GETKEYS (1ULL<<17)  /* Use the modules getkeys interface. */
+#define CMD_MODULE_NO_CLUSTER (1ULL<<18) /* Deny on Redis Cluster. */
```

Comment:
i wish we could make this an `enum`.

Reply:
We could use a macro like getFlag(CMD_MODULE_GETKEYS), where CMD_MODULE_GETKEYS is just a regular enum. The macro would do the bit shifting, since it'll still all be known at compilation time, it should optimize it the same way.

It doesn't seem overly worth it to me.